core/chains/evm/log/registrations.go

```diff
@@ -196,80 +250,83 @@ func filtersContainValues(topicValues []common.Hash, filters [][]Topic) bool {
 	return true
```

Comment:
Returning true from a function that doesn't return anything

Reply:
It returns a boolean.